.github/actions/setup-node-env/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
   install:
     default: true
     description: Whether to install dependencies.
+  downloadLocales:
+    required: false
+    default: "false"
+    description: Whether to download locales artifact.
```

Comment:
Given the majority of uses of this (each CI job that needs them) is having to pass `true`, should we invert the default value of this so we only have to pass it as `false` explicitly in one place?

Reply:
I defaulted to false here to ensure that we don't download the artifact when we don't really need it, especially when we don't even have it. The actions that don't need the locales can run before the artifact has been created, so it should probably make the test suite faster. Besides, `setup-node-env` is used by other workflows, which do not have access to the `locales` artifact.
We could also remove the need for this flag by checking for the artifact's existence and downloading it only if it exists. However, I don't know how to do that, and the actions that do check if the artifact exists seem to be downloading it to do so, which seems inefficient.